Swat/SwatTextareaEditor.php

```diff
@@ -21,6 +21,8 @@ class SwatTextareaEditor extends SwatTextarea
     // }}}
     // {{{ public properties
 
+    public static $tiny_mce_api_key = '';
```

Comment:
I think Mike would prefer a protected static property and then a public static method to set that. Can you do that instead and update the site code to call `SwatTextareaEditor::setTinyMceApiKey('x');`?

Reply:
His comment said a method or property was ok https://silverorange.slack.com/archives/CBSTRPSPP/p1649982320793889?thread_ts=1649962756.957899&cid=CBSTRPSPP